example/src/screens/Home.tsx

```diff
@@ -91,8 +92,9 @@ export function HomeScreen() {
 
   const handleLogin = async () => {
     try {
-      await PointSdkRn.setUserToken(
-        'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkpPeWtVSkFNZjc4M3E2NGtrM0phWCJ9.eyJodHRwczovL2FyZXlvdW9ucG9pbnQuY28vZW1haWwiOiJhbm55QGFlLnN0dWRpbyIsImlzcyI6Imh0dHBzOi8vcG9pbnQtYXBwLWRldi51cy5hdXRoMC5jb20vIiwic3ViIjoiZ29vZ2xlLW9hdXRoMnwxMDIyNTMxNDYxMDA2NDU3Mjk3MDkiLCJhdWQiOlsibG9jYWxob3N0IiwiaHR0cHM6Ly9wb2ludC1hcHAtZGV2LnVzLmF1dGgwLmNvbS91c2VyaW5mbyJdLCJpYXQiOjE2NjkzODczNjYsImV4cCI6MTY2OTQ3Mzc2NiwiYXpwIjoiRmJsRlM1SE02QlNwZlJuSVFCRUdGUllPUWJISFd2eGkiLCJzY29wZSI6Im9wZW5pZCBwcm9maWxlIGVtYWlsIiwicGVybWlzc2lvbnMiOltdfQ.aidfIE5WIJrVcOYifC4a9TO2DgjHAwMA1MRxJb9f_BY8Lf9UzUucaAktcbmS3DgBitQch6CqQETMu2iNbTSXw6jqVjruT3jP1IMoeOXoDRW5qdGI7aT3ROe6HYu3k4LK0RMcJwgWfL10Ozt6pU9LY4RkVkIyq8X9L4o3r2U3fZNR9ln1h4r81bdGRjgQ6N3hNOFTf6YUzQyRZhkW5dDHaJmi6dEdA5zpYZ_-xXfmRegErMl00oNQjvocpPK8d5YJ8c1E-RfiI8YU8P9XdYUCglJdR0PwY43Lxf99Y7dwSi6idTqKMX-LolT7ZovjKiRN8w6CVKaGlfLzLSU8FBJFRQ',
+      await PointSdkRn.setRefreshToken(
+        `${Config.REFRESH_TOKEN}`,
+        `${Config.USER_ID}`
       );
```

Comment:
You probably can call `isOuraAuthenticated();` and `sFitbitAuthenticated();` here before the console.log and after login, so that `fitbitStatus` and `ouraStatus` will have their correct values right after login

Reply:
done